pages/common/ansible-galaxy.md

```diff
@@ -22,3 +22,11 @@
 - Create a new role:
 
 `ansible-galaxy init {{role_name}}`
+
+- Get information about a role:
```

Comment:
Is information about a specific user's role? or system role?

Reply:
User role

Reply:
Perhaps this will read better?
```suggestion
- Get information about a user role:
```

Reply:
@papanito could you apply this change?